src/medigan/model_executor.py

```diff
@@ -146,7 +146,14 @@ def _check_package_resources(self):
                 f"are missing: {e}. Please retry after installing them e.g. via 'pip install "
                 f"{self.dependencies}'."
             )
-            raise e
+            import subprocess
```

Comment:
As discussed, the automatic installation of the packages make it easier for users to quickly run a model. However, we don't know if the users are running medigan in an environment (where the dependencies should be installed) and if the users are fine with the automatic installation (e.g. some packages might not be allowed in a user's setup due to security concerns, licensing, or other reasons). In sum, I think we should at least ask the user via command line if they are fine with the autodownload of [list of packages + versions]

Reply:
I've completely removed this part. Instead, I added a script that allows to install the dependencies for the specified model.